pytest/tests/stress/hundred_nodes/create_gcloud_image.py

```diff
@@ -40,9 +40,11 @@
     sudo apt update
 done
 
-sudo apt install -y python pkg-config libssl-dev build-essential cmake clang llvm
+sudo apt install -y python pkg-config libssl-dev build-essential cmake clang llvm docker.io
+sudo groupadd docker
+sudo usermod -aG docker $USER
 
-curl -sSf https://sh.rustup.rs | sh -s -- -y --default-toolchain nightly-2019-10-04
+curl -sSf https://sh.rustup.rs | sh -s -- -y --default-toolchain nightly-2020-03-19
```

Comment:
can we read the rust-toolchain file so that we don't have to adjust this manually?

Reply:
Yup